com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -55,6 +55,27 @@ public static class RenderPass
             }
         }
 
+        /// <summary>
+        /// This setting controls if the camera editor should display the camera stack category.
+        /// If your renderer is not supporting stacking this one should return 0.
+        /// For the UI to show the Camera Stack widget this must support CameraRenderType.Base.
+        /// <see cref="CameraRenderType"/>
+        /// Returns the bitmask of the supported camera render types in the renderer's current state.
+        /// </summary>
+        public virtual int SupportedCameraRenderTypes()
+        {
+            return 0;
+        }
```

Comment:
Maybe a better API name could help with confusion but not sure at this point. My confusion I think is that each property in the field means something else:
 - A renderer that says a cameraRenderType.Base is supported means that this renderer supports stacking.
 - A renderer that says a cameraRenderType.Overlay is supported means that this renderer can stack on base.

Reply:
We have tried to rename the function to:
`SupportedCameraStackingTypes`

Reply:
@phi-lira